BLSQ/iaso
# POLIO-658: create widget for budget timelines frontend

Explain what problem this PR is resolving

## Self proof reading checklist

## Changes

- create new component `BudgetTimeline` (use of Stepper material ui component)
- Add BudgetTimeline in `BudgetDetails`



## How to test

1. Make sure you have a polio user and the [config](https://drive.google.com/file/d/1SxAVd1uWSzLIqsPzf2Zi4TkU8Kuv51VQ/view?usp=sharing)* to test budget workflow : [POLIO BUDGET WORKFLOW](https://wiki.bluesquare.org/en/Dev-team/product-management/iaso/polio_budget_workflow)
2. Go to Polio > Budget >  Choose a row and click on Budget details 

 ### If you already have a polio account and user:
 Go to Django Admin > Workflow models > Then, add a new model with this config*

### If you don't have a polio account and user : 

[Set up locally via fixtures](https://wiki.bluesquare.org/en/Dev-team/product-management/iaso/polio_budget_workflow#set-up-locally-via-fixtures)


## Print screen / video
![Capture d’écran du 2022-11-03 17-40-59](https://user-images.githubusercontent.com/25134301/199781550-18fec3ac-917d-42ce-a676-95b4dd7adbb1.png)

## Note : 

This [PR](https://github.com/BLSQ/bluesquare-components/pull/89) needs to be merged to see the light green from `stepActive` css class
